### PR TITLE
Adding favicon auto generation to the build process

### PR DESCRIPTION
**Pull request by gauntface**
Concerns:

1.) This only applies in builds, i.e. gulp serve does not run the favicon task since we serve the index.html page.
2.) This happens after HTML minification, meaning that we end up with HTML that is mostly on a single line, but with one block of HTML before the closing head element.

I haven't tested this on devices yet, but looking at the HTML it seems good to go. Happy to do the testing when I get into the London office (likely to be tested on Thursday).

In the mean time - any comments?

### CODE REVIEW

**Comment by addyosmani** [comment]
@gauntface Just a friendly ping as this looks good to land. I'm happy to help with testing.

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "apache-server-configs": "^2.7.1",
     "browser-sync": "^1.3.0",
     "del": "^0.1.3",
+    "favicons": "^1.5.3",
```
Comment (arthurvr) [inline package.json]: Maybe to bump the version? :)

**Comment by gauntface** [comment]
Closing - Rebasing and stuff.

**Comment by sindresorhus** [comment]
No need to close. Just force push to this PR.

**Comment by gauntface** [comment]
I'm going to push to material-sprint rather than master (Can't recall why I pushed to Master in the first place).

**Pull request closed without merging**